### PR TITLE
[WIP] [SearchResults] Add app and route

### PR DESCRIPTION
**Pull request by mzikherman**
EDIT- latest screenshot:
screen shot 2019-03-06 at 6 18 44 pm

This adds the basic page shell for the new search results page. Looks great right?

There's placeholder for the search header (which I was thinking would be the exact/top match which we decided to leave out), the count, the navigation tabs, and then content.

Next up:
  - ~~bringing in the tabs and counts (this requires https://github.com/artsy/gravity/pull/12188 to be merged, and a MP schema update to expose the aggregations)~~
  - bringing in the artwork filter and adding a child route for `/search/artworks`

### CODE REVIEW

File: src/Apps/Search/SearchApp.tsx [hunk 1]
```diff
@@ -0,0 +1,80 @@
...
+
+export class SearchApp extends React.Component<ArtistAppProps> {
+  render() {
+    const { viewer } = this.props
+
+    return (
+      <AppContainer>
+        <HorizontalPadding>
+          <Row>
+            <Col>Search Header</Col>
```
Comment (mzikherman) [inline src/Apps/Search/SearchApp.tsx]: Placeholder, but we probably dont need this (the current designs just have the count displayed and then right into the tabs and individual pages).

File: src/Apps/Search/SearchApp.tsx [hunk 2]
```diff
@@ -0,0 +1,80 @@
...
+      </AppContainer>
+    )
+  }
+}
+
+export const SearchAppFragmentContainer = createFragmentContainer(SearchApp, {
+  viewer: graphql`
+    fragment SearchApp_viewer on Viewer
+      @argumentDefinitions(term: { type: "String!", defaultValue: "" }) {
+      search(query: $term, mode: AUTOSUGGEST, first: 1) {
```
Comment (mzikherman) [inline src/Apps/Search/SearchApp.tsx]: this is `AUTOSUGGEST` for now, the regular site search actually crashes in Gravity b/c its requesting a search thru `MarketingCollection`, but that hasn't been added to the supported list of indexes for site search (only autosuggest). Will update that.

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit 12f2a1cbcd3eb2c58842ea314dbf2f00f75a6dd2

https://deploy-preview-2116--artsy-reaction.netlify.com

**Comment by artsyit** [comment]
Deploy preview for *artsy-reaction* ready!

Built with commit b2479f3b76bd5fb5af22826c75288b613c4b8893

https://deploy-preview-2116--artsy-reaction.netlify.com

File: src/Apps/Search/SearchApp.tsx [hunk 3]
```diff
@@ -0,0 +1,80 @@
...
+import {
+  Footer,
+  RecentlyViewedQueryRenderer as RecentlyViewed,
+} from "Components/v2"
+
+export interface ArtistAppProps {
+  viewer: SearchApp_viewer
+}
+
+export class SearchApp extends React.Component<ArtistAppProps> {
```
Comment (damassi) [inline src/Apps/Search/SearchApp.tsx]: `React.FC` component

**Comment by mzikherman** [comment]
Added the nav tabs:
screen shot 2019-03-06 at 6 18 44 pm

Up next:
  - bringing in artwork filter

**Comment by alloy** [comment]
:ceiling-cat: As per semver, when you _add_ features you should bump the ‘minor’ version. The ‘patch’ component is for fixes to existing features.

**Comment by mzikherman** [comment]
Closing and will re-open in a more condensed form based on the direction we're going in.

**Pull request closed without merging**